adapters/cisco_asa_rest/common.php

```diff
@@ -332,8 +333,9 @@ function scp_to_router($src, $dst)
   $net_profile = get_network_profile();
   $sd = &$net_profile->SD;
   $sd_ip_addr = $sd->SD_IP_CONFIG;
+  $sd_mgt_port = $sd->SD_MANAGEMENT_PORT;
 
-  $ret_scp = exec_local(__FILE__ . ':' . __LINE__, "/opt/sms/bin/sms_scp_transfer -s $src -d $dst_disk:/$dst -l $login -a $sd_ip_addr -p $passwd", $output);
+  $ret_scp = exec_local(__FILE__ . ':' . __LINE__, "/opt/sms/bin/sms_scp_transfer -s $src -d $dst_disk:/$dst -l $login -a $sd_ip_addr -p $passwdi -P $sd_mgt_port", $output);
```

Comment:
$passwdi ==> $passwd

Reply:
fix by this commit https://github.com/openmsa/Adapters/pull/295/commits/38a1123dd0717365f493b27ae08b7561d111d3c8